pkg/render/policyrecommendation.go

```diff
@@ -179,6 +172,21 @@ func (pr *policyRecommendationComponent) clusterRole() client.Object {
 		},
 	}
 
+	if !managedCluster {
```

Comment:
How sure are you about this change? Was the existing RBAC causing a problem?

Reply:
ManagedCluster API is not enabled on a managed cluster and policy recommendation only reads the licensekey from the management cluster.